client-side/framework/vue/app.js

```diff
@@ -0,0 +1,45 @@
+import VueRouter from 'vue-router';
+import Vuex from 'vuex';
+import router from './routes';
+
+require('./bootstrap');
```

Comment:
Would there be any benefit to using vue-bootstrap? https://bootstrap-vue.js.org/docs

Reply:
That bootstrap is referencing the bootstrap file, not the framework.  I use the bootstrap file for pulling in and setting up dependencies IE setting global headers for axios, pulling in lodash, etc.  I do have an example import for pulling in JQuery for bootstrap, but that is just because I know people use Bootstrap framework (I actually don't use bootstrap, I like vuetify or bulma)